services/ui-src/src/components/export/ExportedModalDrawerReportSection.tsx

```diff
@@ -1,15 +1,129 @@
 // components
 import { Box, Heading } from "@chakra-ui/react";
+import { Table } from "components";
 // utils
 import { useStore } from "utils";
 import { ModalDrawerReportPageShape } from "types";
+import { AnyObject } from "yup/lib/types";
```

Comment:
nit: there is an `AnyObject` type you can import from `types` and we should use that instead of the `yup` one

Reply:
Good catch, thanks!